languages/volapük.md

```diff
@@ -5,17 +5,19 @@ nav_order: 999
 nav_exclude: true
 parent: Languages
 layout: language
-title: <code>vo</code>
-description: Machine translation for <code>vo</code>
+title: "Volap\xFCk"
```

Comment:
It's not like this in the .json file.  Maybe something with your Python setup?  We should just use Unicode.

Reply:
And our slugify function for the page name should probably remove diacritics - umlauts, accent marks etc.  languages/volapuk.md not languages/volapük.md

Reply:
> It's not like this in the .json file. Maybe something with your Python setup? We should just use Unicode.

I'll open a separate issue for handling such characters in .md files properly.